index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Document</title>
+    <script type="text/javascript" src="dist/envspec.js"></script>
+     <link rel="stylesheet" href="dist/styles.css">
+  </head>
+
+  <body>
+    <script type="text/javascript" src="dist/demoPage.js"></script>
+    <p><h1>.ENV.SPEC demo page</h1></p>
+    <p><h4><i>Type or paste your .env.spec file in the text area below :</i></h4></p>
+
+    <div>
+        <textarea id="inputArea" placeholder="Start here..." rows = "6" cols = "30"></textarea>
+        <button value="Get HTML" onclick="getHTML()"> Get HTML</button>
+    </div>
+    <div>
```

Comment:
Can you add a heading (I guess `h4`) above here stating that this is `HTML output: `?